### PR TITLE
People list responsivity correction along with some additional tweaks

### PR DESCRIPTION
**Pull request by KrystianZybura**
I made some further changes in people list's responsivity, the effect might not be in 100% aligned with the project origin, but I believe it's much more responsive now instead of just setting the fixed values. Some corrections might be still needed, since the project is enormous and I could've missed few things, so make sure to check if everything is working properly.

Additionally, the new breakpoints that I used for this particular purpose - we might be willing to use them in the future, so I added them globally (it can be adjusted if its an issue) 👌

### CODE REVIEW

File: src/theme.js [hunk 1]
```diff
@@ -13,9 +13,14 @@ export const theme = {
...
   breakpoints: {
+    desktop: "1300px",
     tablet: "1200px",
+    tabletAlternative: "1050px",
+    hugeMobile: "850px",
     defaultMobile: "767px",
+    mediumMobile: "585px",
     smallMobile: "450px",
+    tinyMobile: "320px",
   },
```
Comment (Arcziwald) [inline src/theme.js:24]: Good idea :-)

File: src/common/PersonTile/styled.js [hunk 2]
```diff
@@ -1,61 +1,78 @@
...
 
-  @media (max-width: ${({ theme }) => theme.breakpoints.defaultMobile}) {
-    font-size: 14px;
+  @media (max-width: ${({ theme }) => theme.breakpoints.tablet}) {
+    font-size: 1.8vh;
+  }
+
+  @media (max-width: ${({ theme }) => theme.breakpoints.smallMobile}) {
     margin: 0;
+    font-size: 1.5vh;
```
Comment (WoytekMig) [inline src/common/PersonTile/styled.js]: i suggest to stick to px :) This solution causes problems and it's not necessary ;)

**Review by Arcziwald — APPROVED** [review]
Great job :-)

File: src/theme.js (hunk 1 above)
Comment (WoytekMig) [reply src/theme.js:24]: to be honest, I'm not so sure, it's good to have 8 breakpoints :D  but if it was done like this and it works - I'm not going to coplain :D although I'm quite sure it wasn't necessary ;)

**Review by WoytekMig — APPROVED** [review]
Looks really good, good job ! :)

File: src/common/PersonTile/styled.js (hunk 2 above)
Comment (KrystianZybura) [reply src/common/PersonTile/styled.js]: Fixed, should not cause any problems anymore 😎

**Pull request merged**